Automattic/wp-calypso
# Add/email open graph ultimate

#### Proposed Changes

This PR adds the Email Opens stat graph to the Email Stats page, showing it like this:
<img width="1273" alt="Screenshot 2022-12-28 at 13 15 01" src="https://user-images.githubusercontent.com/3832570/209810897-a0195d52-529a-44df-a879-11b607a86bc8.png">

This is a WIP, and there are something that doesn't work for now, like clicking on a different period on the graph. This will be added in subsequent PRs. This feature is hidden behind the `newsletter/stats` feature flag.

#### Testing Instructions
1. Checkout this branch and start the application.
2. Go to a URL with this format: `calypso.localhost:3000/stats/email/open/[domain-of-your-blog]/day/[post-id]?flags=newsletter/stats`.
3. You should see the stats showing in the graph. Take into consideration that, in order to see stats, an email should have been sent when the post was created, and some followers should have open that email.
4. You can change the URL param `day` to `week` or `month` and you should see how the increments in the chart change accordingly. `year` is still WIP.
